manifests/4.5/logging.openshift.io_clusterlogforwarders_crd.yaml

```diff
@@ -49,22 +80,20 @@ spec:
                   name:
                     description: Name used to refer to the output from a `pipeline`.
                     type: string
-                  secretRef:
-                    description: "SecretRef refers to a `secret` object for secure
-                      communication. \n Client-authenticated TLS is enabled if the
-                      secret contains keys `tls.crt`, `tls.key` and `ca.crt`. Output
-                      types with password authentication will use keys `password`
-                      and `username`, not the exposed 'username@password' part of
-                      the `url`."
+                  secret:
+                    description: "Secret for secure communication. Secrets must be
+                      stored in the namespace containing the cluster logging operator.
+                      \n Client-authenticated TLS is enabled if the secret contains
```

Comment:
nit but should likely be `\nClient-authenticated`

Reply:
Agreed, but it's the CRD generator mangling Go comments so I'm not sure how to fix. The go comment looks fine:
```
	// SecretRef refers to a `secret` object for secure communication.
	//
	// Client-authenticated TLS is enabled if the secret contains keys
	// `tls.crt`, `tls.key` and `ca.crt`. Output types with password
```

Reply:
Agreed, but that's the CRD generator mangling Go comments. Not sure if we can fix it.